tests/dlq_test.go

```diff
@@ -348,6 +348,81 @@ func (s *dlqSuite) validateWorkflowRun(ctx context.Context, run sdkclient.Workfl
 	s.Equal("hello", result)
 }
 
+// This test executes an actual workflow for which we've set up an executor wrapper to return a terminal error. After
+// the workflow task is added to the DLQ, this test then purges the DLQ and call DescribeDLQJob and CancelDLQJob api to verify.
+func (s *dlqSuite) TestPurgeDescribeAndCancel() {
+	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(ctx, testTimeout)
+	defer cancel()
+
+	_, dlqMessageID := s.executeDoomedWorkflow(ctx)
+
+	// Delete the workflow task from the DLQ.
+	token := s.purgeMessages(ctx, dlqMessageID)
+
+	// Verify that the workflow task is no longer in the DLQ.
+	dlqTasks := s.readDLQTasks()
+	s.Empty(dlqTasks, "expected DLQ to be empty after purge")
+
+	// Run DescribeJob and validate
+	response := s.describeJob(token)
+	s.Equal(enums.DLQ_OPERATION_TYPE_PURGE, response.OperationType)
+	s.Equal(enums.DLQ_OPERATION_STATE_COMPLETED, response.OperationState)
+	s.Equal(dlqMessageID, response.MaxMessageId)
+	s.Equal(dlqMessageID, response.LastProcessedMessageId)
+	s.Equal(dlqMessageID+1, response.MessagesProcessed)
```

Comment:
nit: I don't see why this would be related to the message ID--shouldn't it just be 1?